### PR TITLE
Added receptions details on Liebert MPH pdus

### PR DESCRIPTION
**Pull request by RandGH**
Please give a short description what your pull request is for

Add per receptable power draw / voltage etc.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/liebert.yaml [hunk 1]
```diff
@@ -60,8 +66,22 @@ modules:
                     oid: lgpPduPsTable
                     value: lgpPduPsEntryApTotal
                     num_oid: '.1.3.6.1.4.1.476.1.42.3.8.30.20.1.90.{{ $index }}'
-                    descr: 'Apparent Power for {{ $lgpPduPsEntryModel }}'
+                    descr: 'ApPwr {{ $lgpPduPsEntryModel }}'
```
Comment (Jellyfrog) [inline includes/definitions/discovery/liebert.yaml:69]: whats wrong with the old description? ApPwr I have no idea what it means

File: includes/definitions/discovery/liebert.yaml [hunk 2]
```diff
@@ -60,8 +66,22 @@ modules:
...
                     index: 'lgpPduPsEntryApTotal.{{ $index }}'
+                    group: ApparentPower
+                -
+                    oid: lgpPduRcpTable
+                    value: lgpPduRcpEntryPwrOut
+                    num_oid: '.1.3.6.1.4.1.476.1.42.3.8.50.20.1.65.{{ $index }}'
+                    descr: 'Pwr {{ $lgpPduRcpEntryUsrLabel }}'
```
Comment (Jellyfrog) [inline includes/definitions/discovery/liebert.yaml:76]: Power?

File: includes/definitions/discovery/liebert.yaml [hunk 3]
```diff
@@ -72,6 +92,13 @@ modules:
                     index: 'lgpPduRbEntryEcAvailBeforeAlarmHundredths.{{ $index }}'
                     divisor: 100
                     group: Available
+                -
+                    oid: lgpPduRcpTable
+                    value: lgpPduRcpEntryEc
+                    num_oid: '.1.3.6.1.4.1.476.1.42.3.8.50.20.1.60.{{ $index }}'
+                    descr: 'Cur {{ $lgpPduRcpEntryUsrLabel }}'
```
Comment (Jellyfrog) [inline includes/definitions/discovery/liebert.yaml:99]: Current?

File: includes/definitions/discovery/liebert.yaml (hunk 1 above)
Comment (RandGH) [reply includes/definitions/discovery/liebert.yaml:69]: Its too long and getting cut off in the webpage so you can't actually read the receptable name any more

File: includes/definitions/discovery/liebert.yaml (hunk 2 above)
Comment (RandGH) [reply includes/definitions/discovery/liebert.yaml:76]: yes - same issue as above. Adding long prefixes hides the actual device name

File: includes/definitions/discovery/liebert.yaml (hunk 3 above)
Comment (RandGH) [reply includes/definitions/discovery/liebert.yaml:99]: yes

File: includes/definitions/discovery/liebert.yaml (hunk 1 above)
Comment (RandGH) [reply includes/definitions/discovery/liebert.yaml:69]: ok, looks like the current release does not have this issue any more. Will submit an unabbreviated version

**Pull request closed without merging**